docs/csharp/programming-guide/interop/how-to-use-platform-invoke-to-play-a-wave-file.md

```diff
@@ -42,11 +42,10 @@ The following C# code example illustrates how to use platform invoke services to
 4.  Compile and run the code.  
   
 ## .NET Framework Security  
- For more information, see [.NET Framework Security](http://go.microsoft.com/fwlink/?LinkId=37122).  
+ For more information, see [.NET Framework Security](https://technet.microsoft.com/en-us/security/).  
   
 ## See Also  
  [C# Programming Guide](../../../csharp/programming-guide/index.md)  
  [Interoperability Overview](../../../csharp/programming-guide/interop/interoperability-overview.md)  
- [Interoperability Overview](../../../csharp/programming-guide/interop/interoperability-overview.md)  
- [A Closer Look at Platform Invoke](http://msdn.microsoft.com/library/ba9dd55b-2eaa-45cd-8afd-75cb8d64d243)  
+ [A Closer Look at Platform Invoke](https://docs.microsoft.com/en-us/dotnet/framework/interop/consuming-unmanaged-dll-functions#a-closer-look-at-platform-invoke)  
```

Comment:
good job finding the replacement here. Let's just do a relative link such as:
consuming-unmanaged-dll-functions.md#a-closer-look-at-platform-invoke